ui/src/components/spinner/spinner-dot.tsx

```diff
@@ -0,0 +1,28 @@
+import { SpinnerStyles } from './spinner.styles';
+
+export const SpinnerDot: React.FC<SpinnerStyles.ContainerProps> = (props) => {
+  return (
+    <>
+      <style>
+        {`
+        .spinner_Wezc{transform-origin:center;animation:spinner_Oiah .75s step-end infinite}@keyframes spinner_Oiah{8.3%{transform:rotate(30deg)}16.6%{transform:rotate(60deg)}25%{transform:rotate(90deg)}33.3%{transform:rotate(120deg)}41.6%{transform:rotate(150deg)}50%{transform:rotate(180deg)}58.3%{transform:rotate(210deg)}66.6%{transform:rotate(240deg)}75%{transform:rotate(270deg)}83.3%{transform:rotate(300deg)}91.6%{transform:rotate(330deg)}100%{transform:rotate(360deg)}}
+        `}
+      </style>

```

Comment:
These could be moved to stitches keyframes and applied over `spiner.styles.ts` styling file.